php/api/controller/class-audit-posts-controller.php

```diff
@@ -659,19 +659,22 @@ public function get_items_permissions_check( $request ) {
 	public function check_read_permission( $post ) {
 		$allowed = parent::check_read_permission( $post );
 
-		/**
-		 * Determine an item's visibility.
-		 * Still allowed users with the right capabilities to read this post.
-		 */
+		// Determine an item's visibility and allow users with the right capabilities to read this post.
 		$visibility   = get_post_meta( $post->ID, 'visibility', true );
 		$current_user = wp_get_current_user();
-		$is_author    = $post->post_author === $current_user->ID && in_array( 'api_client', (array) $current_user->roles, true );
```

Comment:
@valendesigns wouldn’t `$current_user->has_cap( 'api_client' )` be better than `in_array( 'api_client', (array) $current_user->roles, true )` ?

Reply:
Good call, changed!

Reply:
:)